<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>definite_teacher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);</string>
  </property>
  <widget class="QComboBox" name="combo_teacher_full_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ФИО учителя</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Класс</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Здание</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Предмет</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Часы</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_hours">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общее количество часов:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_choice_teacher">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_class">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Класс</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_subject">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Предмет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_del_load">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/pngwing.com (11).png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
